### PR TITLE
support Filament V3

### PR DESCRIPTION
**Pull request by LarsCaspers**
Hi Saade,

This pull request adds compatibility for Filament V3.
Please let me know if i need to change anything.

### CODE REVIEW

File: src/Pages/ViewLog.php [hunk 1]
```diff
@@ -14,13 +14,14 @@ class ViewLog extends Page
 {
     use HasSecurity;
 
-    protected static string $view = 'filament-laravel-log::view-log';
+    public static string $view = 'filament-laravel-log::view-log';
```
Comment (nguyentranchung) [inline src/Pages/ViewLog.php:17]: why this file all protected method become public

File: src/Pages/ViewLog.php [hunk 2]
```diff
@@ -14,13 +14,14 @@ class ViewLog extends Page
...
 
     public ?string $logFile = null;
 
     public function refreshContent(): void
     {
-        $this->dispatchBrowserEvent('logContentUpdated', ['content' => $this->readLog()]);
+        $content = $this->readLog();
+        $this->dispatch('logContentUpdated', ['content' => $content]);
```
Comment (nguyentranchung) [inline src/Pages/ViewLog.php:24]: $this->dispatch('logContentUpdated', ['content' => $this->readLog()]);

just one line

File: src/Pages/ViewLog.php [hunk 3]
```diff
@@ -68,42 +69,42 @@ protected function getForms(): array
...
+    public function getFormSchema(): array
     {
         return [
             Forms\Components\Select::make('logFile')
                 ->searchable()
                 ->reactive()
                 ->disableLabel()
                 ->placeholder(__('log::filament-laravel-log.forms.search.placeholder'))
-                ->options(fn () => $this->getFileNames($this->getFinder())->take(5))
+                ->options(fn () => $this->getFileNames($this->getFinder())->take(10))
```
Comment (nguyentranchung) [inline src/Pages/ViewLog.php:80]: this PR for support Filament V3, dont change!

File: resources/views/view-log.blade.php [hunk 4]
```diff
@@ -82,20 +81,17 @@ class="inline-flex items-center justify-center px-2 font-medium tracking-tight t
...
+        <x-filament::button type="button" x-on:click="isOpen = false" color="secondary" outlined="true"
+                            class="filament-page-modal-button-action">
+            {{ __('log::filament-laravel-log.modals.clear.actions.cancel') }}
+        </x-filament::button>
+        <x-filament::button wire:click="clearLogs" x-on:click="isOpen = false" type="button" color="primary"
+                            class="filament-page-modal-button-action">
+            {{ __('log::filament-laravel-log.modals.clear.actions.confirm') }}
+        </x-filament::button>
     </x-filament::modal>
+
```
Comment (nguyentranchung) [inline resources/views/view-log.blade.php:96]: respect format

File: resources/views/view-log.blade.php [hunk 5]
```diff
@@ -82,20 +81,17 @@ class="inline-flex items-center justify-center px-2 font-medium tracking-tight t
         <div x-ref="editor" class="mt-4 rounded-md ace-forge" />
     </div>
 
-    <x-filament::modal id="filament-laravel-log--clear-modal"
+     <x-filament::modal id="filament-laravel-log--clear-modal"
```
Comment (nguyentranchung) [inline resources/views/view-log.blade.php:84]: look line indent wrong

**Comment by saade** [comment]
Hi! @LarsCaspers and @nguyentranchung, I've planned a bit more for v3. So I'm closing this in favor of https://github.com/saade/filament-laravel-log/pull/24

Thank you for contributing!

**Pull request closed without merging**